src/lib389/lib389/utils.py

```diff
@@ -1701,10 +1708,36 @@ def is_valid_hostname(hostname):
     return all(allowed.match(x) for x in hostname.split("."))
 
 
+def parse_size(size):
```

Comment:
Could you please add docstrings for the added functions, describing what they do. Like here, and `def get_default_mdb_max_size(paths):`.

Reply:
Done